<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bank</class>
 <widget class="QMainWindow" name="Bank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_wellcome">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>سلام ، به بانک آینده روشن 🏦 خوش آمدید.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>خـــروج</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_god">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-- به نام بی نیاز عالم آرا --</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>421</width>
      <height>471</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="employee_2">
     <attribute name="title">
      <string>کارمند</string>
     </attribute>
     <widget class="QGroupBox" name="box_username_pass_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>ورود </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <widget class="QLabel" name="label_username_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>نام کاربری:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_password_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>گــــــذرواژه:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_username_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_password_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="button_login_employee_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ورود</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>لطفا نام کاربری و رمز عبور خود را وارد کنید.</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="customer_2">
     <attribute name="title">
      <string>مشتری</string>
     </attribute>
     <widget class="QGroupBox" name="box_username_pass_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>ورود </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <widget class="QLabel" name="label_username_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>نام کاربری:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_password_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>گــــــذرواژه:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_username_customer">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_password_customer">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="button_login_customer">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ورود</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>لطفا نام کاربری و رمز عبور خود را وارد کنید.</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_wellcome_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>نام شعبه:                                    کد شعبه:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="branchname">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="branch_code">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_exit</sender>
   <signal>clicked()</signal>
   <receiver>Bank</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>655</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
